src/isar/apis/telemetry/mission_status.py

```diff
@@ -1,67 +1,67 @@
-import logging
-from http import HTTPStatus
+# import logging
+# from http import HTTPStatus
 
```

Comment:
Delete file entirely. Old functionality like this will become technical debt if it's left like this. If we need the code for future reference we can always go back in the git history and see what was there when we need it.